components/NavigationBar/src/MDCNavigationBar.h

```diff
@@ -99,18 +99,11 @@ IB_DESIGNABLE
 @property(nonatomic, strong, nullable) UIView *titleView;
 
 /**
- Display attributes for the titleView's title text.
-
- Setting this property will render an NSAttributedString with the assigned attributes across the
- entire text.
+ The font applied to the title of navigation bar.
+ Font size is enforced to 20.
```

Comment:
Are we sure this is absolutely necessary?

Reply:
I actually don't have an strong opinion on this. @ianegordon do you think we should allow users to set any font, leaving it to them to make sure it looks correct with their font?

Reply:
Comments keep getting lost, so reposting here:

I've filed https://www.pivotaltracker.com/story/show/156482550 to discuss this further. I also agree that the intention to fix to a specific font size is a bit surprising, but this PR is not changing the behavior. Any fix or change to that behavior can and should be done in a follow-up PR.